zerver/tests/test_reactions.py

```diff
@@ -573,84 +573,81 @@ def test_delete_non_existent_zulip_reaction(self) -> None:
         self.assert_json_error(result, "Reaction doesn't exist.")
 
 class RealmEmojiReactionTests(EmojiReactionBase):
+    def setUp(self) -> None:
+        green_tick_emoji = RealmEmoji.objects.get(name="green_tick")
+        self.default_reaction_info = {
+            'emoji_name': 'green_tick',
+            'emoji_code': str(green_tick_emoji.id),
+        }
+
     def test_add_realm_emoji(self) -> None:
+        result = self.post_reaction(self.default_reaction_info)
+        self.assert_json_success(result)
+
+    def test_non_numeric_emoji_code(self) -> None:
         reaction_info = {
             'emoji_name': 'green_tick',
-            'emoji_code': 'green_tick',
+            'emoji_code': 'non_existent',
```

Comment:
shouldn't this be a number, e.g. `123456`?  Seems more readable.

Reply:
That test was there for specifically testing non-numeric emoji codes but I again checked the code and neither the code nor the test makes any sense now. So I removed both of them.